src/main/java/dev/codesupport/web/api/data/repository/ShowcaseRepository.java

```diff
@@ -4,9 +4,11 @@
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.stereotype.Repository;
 
+import java.util.List;
+
 @Repository
 public interface ShowcaseRepository extends JpaRepository<ShowcaseEntity, Long> {
 
-
+    List<ShowcaseEntity> findAllByUser_IdOrderById(Long userId);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The instance method name 'findAllByUser_IdOrderById' doesn't match '[a-z][a-zA-Z0-9]*'](https://app.codacy.com/gh/codesupport/website-backend/pullRequest?prid=5323316)